clients/js/src/revisions/v2/additionalSigner.ts

```diff
@@ -8,6 +8,8 @@ import {
 } from '@metaplex-foundation/umi';
 import { wrapSerializerInRuleHeaderV2 } from './ruleHeader';
 import { RuleTypeV2 } from './ruleType';
+import { RuleV1 } from '../v1';
```

Comment:
Same here when importing `RuleV1` into the v2 folder for all RuleV2 instances.

```suggestion
import type { RuleV1 } from '../v1';
```